src/frontend/src/styles/main.css

```diff
@@ -156,9 +157,9 @@ button {
   margin-bottom: 1rem;
   padding: 0.6rem 1rem;
   border-radius: 4px;
-
   line-height: 1.25rem;
   letter-spacing: 0.005em;
+  font-family: Arial;
```

Comment:
I know that it was `Arial` before. But I think it look nicer if its the same font then the rest of the site no? I think it was an omission before.

Reply:
Yup, sure it seems to look great with same font. I might have to adjust certain button font-size, but that should be simple enough!

<image src="https://user-images.githubusercontent.com/38238542/182981797-ea9c3612-4302-40fc-91b8-4a971cbf5f67.jpeg" width=400>